doc/tutorials/app/orbbec_astra.markdown

```diff
@@ -18,7 +18,7 @@ camera interface.
 
 In order to use the Astra camera's depth sensor with OpenCV you should do the following steps:
 
--#  Download the latest version of Orbbec OpenNI SDK (from here <https://orbbec3d.com/index/download.html>).
+-#  Download the Orbbec OpenNI SDK (from here https://3dclub.orbbec3d.com/t/universal-download-thread-for-astra-series-cameras/622).
```

Comment:
The SDK can be accessed via Orbbec's 3D club website. We can use the link instead of GDrive.

Reply:
I propose to mention here about official download and why we propose alternative link.